### ISSUE TITLE
[Feat]: Improve Docker image usability by adding WORKDIR /etc/netdata

### ISSUE THREAD

**Issue by powerman** [netdata/netdata#17890]
### Problem

Most of commands executed inside a docker container are `./edit-config`, which must be run in `/etc/netdata`. So, each time after `docker exec` we have to run `cd /etc/netdata`, which is a bit annoying.

### Description

Make `/etc/netdata` default directory inside docker container.

### Importance

nice to have

### Value proposition

1. Less commands to run each time we need to reconfigure netdata in a docker.
2. Less errors because of forgotten `cd`.
3. Better overall UX for docker deployments. :smile: 

### Proposed implementation

Add `WORKDIR /etc/netdata` near the bottom of the `Dockerfile`.

**Comment by ilyam8** [comment]
Hey, @powerman. Please open one issue with all your suggestions.

**Comment by ilyam8** [comment]
All of them are about "Install Netdata with Docker" documentation. You can open one issue and list all your suggestions. Thanks!

**Comment by ilyam8** [comment]
I'm sorry, I meant #17892, #17891, and #17893. Not this one.

@Ferroin what do you think about this request?

**Comment by Ferroin** [comment]
The question comes down to whether the entrypoint script and the agent itself actually correctly handle being run in `/etc/netdata` instead of `/`. That said, if `docker exec /etc/netdata/edit-config` does not work but opening a shell, changing directory, and running `edit-config` does, that’s a bug in the edit-config script.

**Comment by powerman** [comment]
Hmm. To me it's a different thing. `docker exec /etc/netdata/edit-config` works okay. But with `WORKDIR` set `docker exec ./edit-config` will work.

**Comment by ilyam8** [comment]
Should we just `ln -s /etc/netdata/edit-config /usr/bin/`? Will it do?

**Comment by powerman** [comment]
Good idea! But it won't make WORKDIR change useless - when in a container if you're in `/etc/netdata` then you can use shell auto-complete for existing config files. So, to me both adding symlink into `/usr/bin` (or, better, `/usr/local/sbin`) and `WORKDIR /etc/netdata` will provide best UX.

**Comment by powerman** [comment]
FWIW Here is one-liner example to debug plugins in docker:
```sh
sudo -u netdata -EH env -C /usr/libexec/netdata/plugins.d ./go.d.plugin -d -m nvidia_smi
```
Maybe worth to add it somewhere in the docs too.
